doc/CCPPtechnical/source/CompliantPhysicsParams.rst

```diff
@@ -112,10 +91,76 @@ More details are found below:
 
 * Interstitial modules (``scheme_pre`` and ``scheme_post``) can be included if any part of the physics
   scheme must be executed before (``_pre``) or after (``_post``) the ``module scheme`` defined above.
-  These situations are described in more detail in :numref:`Section %s <DynamicBuildCaps>`.
 
 .. _IOVariableRules:
 
+.. _MetadataRules:
+
+Metadata Rules
+==============
+* Metadata files (``.meta``) are in XML format and contain metada for one or more CCPP entry point schemes.
+  There should be one .meta file for each ``.f`` or .``F90`` file.
+
+* For each CCPP compliant scheme, the ``.meta`` file should have this set of lines
+
+.. code-block:: fortran
+
+   [ccpp-arg-table]
+   name =
+   type =
+
+* ``ccpp-arg-table`` indicates the start of a new metadata section for a given scheme.
+
+* ``name`` is name of the corresponding subroutine/module.
+
+* ``type`` can be ``scheme``, ``module``, ``DDT``, or ``host``.
+
+* For empty schemes, the three lines above are sufficient. For non-empty schemes, the metadata should
+  describe all input and output arguments to the scheme using the following format:
+
+.. code-block:: fortran
+
+   [varname]
+    standard_name =
+    long_name = 
+    units =
+    rank = 
+    dimensions =
+    type =
+    kind =
+    intent =
+    optional =
+
+* The following attributes are optional: ``long_name``, ``kind``, and ``optional``.
+
+* Lines can be combined using | as a separator, e.g.,
+
+.. code-block:: fortran
+
+   type = real | kind = kind_phys
+
+* ``varname`` is the local name of the variable in the subroutine.
+
+* The dimensions attribute should be empty for scalars or contain the ``standard_name`` for the start and end for
+  each dimension of an array. Default is 1, For example:
+
+.. code-block:: fortran
+
+   dimensions = ()
+   dimensions = (ccpp_constant_one:horizontal_loop_extent, vertical_level_dimension)
+   dimensions = (horizontal_dimension,vertical_dimension)
+   dimensions = (horizontal_dimension,vertical_dimension_of_ozone_forcing_data,number_of_coefficients_in_ozone_forcing_data)
```

Comment:
It could be mentioned here that `ccpp_constant_one` is the assumed start for any dimension which only has a single value.

Reply:
Changed from 
Default is 1.
to
ccpp_constant_one is the assumed start for any dimension which only has a single value.